test/Api.IntegrationTest/SecretsManager/Controllers/ServiceAccountsControllerTests.cs

```diff
@@ -943,6 +929,22 @@ private async Task<(Guid OrganizationId, List<Guid> ServiceAccountIds)> SetupLis
         return (org.Id, serviceAccountIds);
     }
 
+    private static ApiKey CreateTestApiKey(Guid serviceAccountId, string name = "Test Name", DateTime? expiresAt = null)
```

Comment:
```suggestion
    private static ApiKey CreateTestApiKey(Guid serviceAccountId, string name = _mockEncryptedString, DateTime? expiresAt = null)
```
Name field is always encoded, this is better default